packages/bpk-component-mobile-scroll-container/src/BpkMobileScrollContainer.js

```diff
@@ -102,11 +105,19 @@ class BpkMobileScrollContainer extends Component<Props, State> {
   componentDidMount() {
     this.setScrollBarAwareHeight();
     this.setScrollIndicatorClassName();
-    window.addEventListener('resize', this.onWindowResize);
+    this.scrollerEl.addEventListener('resize', this.onWindowResize);
+    this.scrollerEl.addEventListener('mousemove', this.performDrag);
+    this.scrollerEl.addEventListener('mousedown', this.enableDragging);
+    this.scrollerEl.addEventListener('mouseup', this.disableDragging);
+    this.scrollerEl.addEventListener('mouseleave', this.disableDragging);
```

Comment:
We do this on `mouseleave` too, so that if a user starts clicking and dragging but then leaves the `scrollerEl` before `mouseup`, dragging stops.